src/components/node-list/index.js

```diff
@@ -34,10 +31,17 @@ class NodeList extends React.Component {
    * @return {object} The results array with a new field added
    */
   highlightMatch(results) {
-    return results.map(result => ({
-      highlightedLabel: getHighlightedText(result.name, this.state.searchValue),
-      ...result
-    }));
+    const obj = {};
+    Object.keys(results).forEach(key => {
+      obj[key] = results[key].map(result => ({
```

Comment:
Should this guard against results[key] not being an array?

Reply:
I don't think there's ever a case where it wouldn't be. I can add a check but I'm not sure that it's necessary

Reply:
👍